src/rules/selector-no-redundant-nesting-selector/README.md

```diff
@@ -59,3 +59,51 @@ p {
   }
 }
 ```
+
+## Options
+
+Some false alarms can be blocked by assigning keywords:
+
+```js
+{
+  rules: {
+    'scss/selector-no-redundant-nesting-selector', [true, { keywords: ['when'] }],
+  },
+}
+```
+
+The following patterns are *not* considered warnings:
```

Comment:
There could be some kind of additional text here to say that this is Less syntax.